### DISCUSSION TITLE
How to get underlined links back in AnuPpuccin

### DISCUSSION THREAD

**Discussion by OpenSaucier** [AnubisNekhet/AnuPpuccin#263] [Snippets]
After installing and enabling the theme, the underlining of [[links]] have disappeared.
Is there some way I can retoggle them or enable any css snippets?
![Screenshot 2024-04-07 at 2 46 01 AM](https://github.com/AnubisNekhet/AnuPpuccin/assets/166250414/9c01e50a-075c-49cf-ae16-a4c9dfc6ad5e)

**Comment by AaDeChem** [comment]
1. Navigate to the folder "YOUR_VAULT_NAME/.obsidian/themes/AnuPpuccin"
2. open the file "theme.css" in an editor
3. search the following settings:
 
`.cm-hmd-internal-link {   --link-decoration: none;   
--link-decoration-hover: none;   
--link-decoration-active: none; }`

change it to:

`.cm-hmd-internal-link {   --link-decoration: underline;   
--link-decoration-hover: none;   
--link-decoration-active: none; }`
